gst/nnstreamer/include/nnstreamer_plugin_api_trainer.h

```diff
@@ -0,0 +1,144 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * NNStreamer API for Tensor_Trainer Sub-Plugins
+ * Copyright (C) 2022 Hyunil Park <hyunil46.park@samsung.com>
+ */
+/**
+ * @file  nnstreamer_plugin_api_filter.h
+ * @date  1 Dec 2022
+ * @brief Mandatory APIs for NNStreamer Trainer sub-plugins (No External Dependencies)
```

Comment:
What does ```(No External Dependencies)``` mean precisely?

Reply:
This term is also used for tensor_plugin_api_filter.h
It seems that this header file has no external dependencies.
Thank you.